src/core/tooltips/rfc.md

```diff
@@ -0,0 +1,86 @@
+## Tooltip Service
+
+This service allows us to show tooltips in different places of our application.
+Underhood it uses `@konturio/ui-kit` tooltip component for show tooltip in theme style
+
+Tooltip:
+
+- Shows by hover-in and hides by hover-out events (after delay)
+- Not interactive
+- Can contain only text
+
+## How to use
+
+We assume that all cases of using a tooltip can be reduced to two cases
+
+### We have an element around which a tooltip should be shown
+
+In that case you need to import special components and use them
+
+```jsx
+import { Tooltip, TooltipTrigger, TooltipContent } from '~core/tooltip';
+
+function Sidebar() {
+  return (
+    <nav>
+      <Tooltip>
```

Comment:
According to the design, the tooltip can be 2 sized, regular and small, we need some prop for the sizing
<img width="924" alt="image" src="https://user-images.githubusercontent.com/14041229/220108365-09756ff7-ac41-4212-a014-2fe03ad1b8e8.png">